src/main/java/bl4ckscor3/mod/isamodbyvazkii/IsAModByVazkii.java

```diff
@@ -37,16 +37,13 @@ public void postInit(FMLPostInitializationEvent event)
 		{
 			String authorList = modList.get(modid).getMetadata().getAuthorList();
 
-			if(authorList.toLowerCase().contains("vazkii")) 
-				MODS.put(modid, modList.get(modid).getName());
-
```

Comment:
Is this removal intended? I believe it won't work trying to get any keys/values if you don't populate the Map 😅

Reply:
oh right, need it for items/blocks

sorry lol